.github/workflows/build-docker-images.yml

```diff
@@ -0,0 +1,115 @@
+name: build-docker-images
+
+on:
+  pull_request:
+#  workflow_dispatch:
+
+env:
+  DOCKER_BUILDKIT: 1
+  REGISTRY: docker.elastic.co
+  PREFIX: observability-ci
+
```

Comment:
the permissions section is not declared

```yaml
permissions: 
  content: read
```

Reply:
https://github.com/elastic/apm-pipeline-library/pull/1986/commits/b77b55cb58beea30649838c4ac4fa344ba8f4156